### ISSUE TITLE
Subscription do not existed

### ISSUE THREAD

**Issue by xiaocode** [KevinEdry/nestjs-trpc#3]
I can not find @Subscription in package

**Comment by KevinEdry** [comment]
Yeah, I haven't implemented it yet, I wanted to test that the initial implementation is stable before going and implementing the subscription feature.
Is that a critical feature for your use-case?

**Comment by xiaocode** [comment]
Yeah, I want to use websocket in nest.js with trpc, so Subscription is necessary.

**Comment by KevinEdry** [comment]
I'll try and figure out how to incorporate the two then.

**Comment by KevinEdry** [comment]
@xiaocode it seems like `trpc` is using `ws` as per their docs [here](https://trpc.io/docs/v10/subscriptions).
I am thinking of going with the nest approach here for abstraction with the implementation to look something like this:
```typescript
 @Subscription("eventName", {
   input: z.object({ userId: z.string() }),
   output: z.string(),
 })
 async handleEvent(
   @Input('userId') userId: string,
 ): string {
  return userId
 }
```
Is this something compelling?

**Comment by xiaocode** [comment]
It`s OK!

**Comment by KevinEdry** [comment]
Hey @xiaocode, subscriptions will come when trpc `v11` will go out of alpha.

**Comment by XBeg9** [comment]
Any updates on this?
